soccer/Robot.cpp

```diff
@@ -162,8 +161,10 @@ void OurRobot::resetForNextIteration() {
 }
 
 void OurRobot::resetMotionConstraints() {
+    _rotationConstraints = Planning::RotationConstraints();
     _motionConstraints = MotionConstraints();
-    _motionCommand = Planning::MotionCommand();
+    _motionCommand = make_unique<Planning::WorldVelTargetCommand>(Geometry2d::Point(0,0));
```

Comment:
You're right that "None" is sort of functionally equivalent to a world vel of zero, but I think having them separate makes it a little cleaner.  If we wanted to print out the inputs to the path-planning layer, it would be a lot nicer to see "None" or `nullptr` for a command and know that the gameplay system never set it to anything rather than seeing a worldvel of zero.  Would it be tough to use `nullptr` and not have a "None" type?